AndelaOSP/andela-societies-frontend
# Resolves #54 Create the Redemptions Page

Resolves #54
Description
This gives users the ability to access the redemptions page that display recent redemptions

Fix
Add the Redemptions page component
Add the ActivityCard.jsx for the Redemptions page

How to test

- Pull this branch if you don't have it already. (git pull origin ft-user-should-be-able-to-logout-155808774).
 - Run yarn start. Visit the app on the browser, the SignIn page should appear
- Once you are logged in, click on the `Redemption` link on the sidebar menu
- It should redirect you to the redemptions page. Check screenshot below for clarification

<img width="1440" alt="screen shot 2018-04-09 at 3 30 57 am" src="https://user-images.githubusercontent.com/26271143/38476592-77ebcf6a-3ba6-11e8-90b1-22fd33bab784.png">
